app/webpacker/components/Tickets/TicketWorkbenches/AnonymizationTicketWorkbenchForWrt/index.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Container, Header, Message } from 'semantic-ui-react';
+import { QueryClient, useQuery } from '@tanstack/react-query';
+import getDetailsBeforeAnonymization from './api/getDetailsBeforeAnonymization';
+import Loading from '../../../Requests/Loading';
+import Errored from '../../../Requests/Errored';
+import VerifyAnonymizeDetails from './VerifyAnonymizeDetails';
+import ReviewSystemGeneratedChecks from './ReviewSystemGeneratedChecks';
+import PerformManualChecks from './PerformManualChecks';
+import AnonymizeAction from './AnonymizeAction';
+
+const ANONYMIZATION_QUERY_CLIENT = new QueryClient();
+
+export default function AnonymizationTicketWorkbenchForWrt({ userId, wcaId }) {
+  const {
+    data, isLoading, isError,
+  } = useQuery({
+    queryKey: ['anonymizeDetails', userId, wcaId],
+    queryFn: () => getDetailsBeforeAnonymization(userId, wcaId),
+    enabled: Boolean(userId || wcaId),
+  }, ANONYMIZATION_QUERY_CLIENT);
+
+  if (isLoading) return <Loading />;
+  if (isError) return <Errored />;
+
+  if (!userId && !wcaId) {
```

Comment:
The important thing is that `data` exists, right? (Which is equivalent, given the `enabled` value above, but that could change in the future.)
```suggestion
  if (!data) {
```

Reply:
Yes you are right, changed.